server/app/services/export/PdfExporter.java

```diff
@@ -220,6 +254,182 @@ private byte[] buildApplicationPdf(
+   *     questions nested under an enumerator, 2 for doubly-nested enumerator questions, etc. Should
+   *     be multiplied by {@code INDENTATION_PER_LEVEL} when adding text into the PDF.
+   */
+  private void renderProgramBlock(
+      Document document,
+      ProgramDefinition program,
+      BlockDefinition block,
+      ImmutableList<QuestionDefinition> allQuestions,
+      int indentationLevel)
+      throws DocumentException {
+    document.add(Chunk.NEWLINE);
+    LineSeparator ls = new LineSeparator();
+    ls.setAlignment(Element.ALIGN_RIGHT);
+    // Approximately indent the line separator by subtracting 5% of its width per level of
+    // indentation.
+    ls.setPercentage(100 - (indentationLevel * 5));
+    document.add(ls);
+    document.add(Chunk.NEWLINE);
+
+    // Block-level information
+    document.add(text(block.name(), H2_FONT, indentationLevel));
+    document.add(
+        text("Admin description: " + block.description(), SMALL_GRAY_FONT, indentationLevel));
+    document.add(Chunk.NEWLINE);
+
+    // Visibility & eligibility information
+    if (block.visibilityPredicate().isPresent()) {
+      renderPredicate(
+          document, block.visibilityPredicate().get(), block, allQuestions, indentationLevel);
+    }
+    if (block.eligibilityDefinition().isPresent()) {
+      renderPredicate(
+          document,
+          block.eligibilityDefinition().get().predicate(),
+          block,
+          allQuestions,
+          indentationLevel);
+    }
+
+    for (int i = 0; i < block.getQuestionCount(); i++) {
+      // Question-level information
+      QuestionDefinition question = block.getQuestionDefinition(i);
+      document.add(text(question.getQuestionText().getDefault(), H3_FONT, indentationLevel));
+      if (!question.getQuestionHelpText().isEmpty()) {
+        document.add(
+            text(question.getQuestionHelpText().getDefault(), PARAGRAPH_FONT, indentationLevel));
+      }
+      document.add(text("Admin name: " + question.getName(), SMALL_GRAY_FONT, indentationLevel));
+      document.add(
+          text(
+              "Admin description: " + question.getDescription(),
+              SMALL_GRAY_FONT,
+              indentationLevel));
+      document.add(
+          text(
+              "Question type: " + question.getQuestionType().name(),
+              SMALL_GRAY_FONT,
+              indentationLevel));
+
+      // If a question offers options, put those options in the PDF
+      if (question.getQuestionType().isMultiOptionType()) {
+        MultiOptionQuestionDefinition multiOption = (MultiOptionQuestionDefinition) question;
+        List list = createList(indentationLevel);
+        for (QuestionOption option : multiOption.getOptions()) {
+          list.add(new ListItem(option.optionText().getDefault()));
+        }
+        document.add(list);
+      } else if (question.getQuestionType() != QuestionType.STATIC) {
```

Comment:
Since this is generating an overview of the program configuration we probably want to include the static questions.